src/intTest/resources/application.yml

```diff
@@ -118,4 +118,6 @@ nhais:
       3WJ4bVSmp3MOQ2EF4zrAl4fXEd4lS3JxKtzbHFbSogAmPFnuP8wIDWTDgfEq
       -----END RSA PRIVATE KEY-----}
     recipientCodes: ${NHAIS_MESH_RECIPIENT_CODES:XX11=A91561OT001
-      TES5=A91561OT002}
\ No newline at end of file
+      TES5=A91561OT002}
+  scheduler:
+    intervalInSeconds: ${NHAIS_SCHEDULER_INTERVAL_IN_SECONDS:5}
```

Comment:
This variable is used in different context than the name says.
I think there should be 2 vars here:

```
interval: 60000 # milliseconds
overlap_interval: 5000 # milliseconds
```